python/ray/log.py

```diff
@@ -0,0 +1,158 @@
+import logging
```

Comment:
Hmm can you at least move this to _private repo? Or is it also difficult?

Reply:
Nope, there's no problem with doing this - as long as there are no calls to `import ray` inside the subpackage at initialization time, which there aren't. I've moved the file there now.